### DISCUSSION TITLE
Snowlake to Elasticsearch connector timezone

### DISCUSSION THREAD

**Discussion by fabiobranis** [airbytehq/airbyte#34516] [Connector Questions]
Hello, Airbyte community.
This is my first issue here.
I've been facing an issue here related to the timezone. 
I'm on UTC-3 time zone and my applications work on UTC timezone. Our Airbyte instance is hosted on AWS, which works in the UTC time zone.
Our snowflake database is set up to work in the UTC timezone as well.
Also, I've been setting up the session of my query to retrieve dates in the UTC time zone.
With all that being said, everything works well until Airbyte cron triggers.
All the logs are UTC (in a container), but the time zone, IDK which is the right one, since my day only turns at 10 AM UTC-3 (honestly, IDK a timezone that can match this).
So, how can we set the right timezone for Airbyte self-hosted?

**Comment by sajarin** [comment]
I think Docker containers by default run in UTC time, are you deploying using Docker? There might be a way to be able to run the docker container in a different timezone. Could you clarify what timezone you'd like to set Airbyte to? Is it UTC-3 or UTC?

**Comment by fabiobranis** [comment]
Hi.
Thanks for the reply.
The thing is: I can't tell the timezone, it's very strange because the
query just turns to the other day after 10AM in UTC-3. So, I can't infer
which timezone is the container. We've made some configurations here, all
unsuccessful. But I'm going to talk with the Devops guys about this docker
thing and then I'll get back to you.
Thanks a lot.

Em sex., 16 de fev. de 2024 às 14:31, Sajarin ***@***.***>
escreveu:

> I think Docker containers by default run in UTC time, are you deploying
> using Docker? There might be a way to be able to run the docker container
> in a different timezone. Could you clarify what timezone you'd like to set
> Airbyte to? Is it UTC-3 or UTC?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/airbytehq/airbyte/discussions/34516#discussioncomment-8495353>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABU2DDQO45V2KMUOOCR4QFTYT6JXDAVCNFSM6AAAAABCKPZRWKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DIOJVGM2TG>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Fábio de Siqueira Branis Nunes*
Celular: (48) 9989-7667
E-mail: ***@***.***

[a comment by an automated account is not shown]
